src/init/common.cpp

```diff
@@ -81,7 +81,7 @@ void AddLoggingArgs(ArgsManager& argsman)
 void SetLoggingOptions(const ArgsManager& args)
 {
     LogInstance().m_print_to_file = !args.IsArgNegated("-debuglogfile");
-    LogInstance().m_file_path = AbsPathForConfigVal(fs::PathFromString(args.GetArg("-debuglogfile", DEFAULT_DEBUGLOGFILE)));
+    LogInstance().m_file_path = AbsPathForConfigVal(args.GetPathArg("-debuglogfile", DEFAULT_DEBUGLOGFILE));
```

Comment:
693cdca5f552f74de3665a874ce6acf6c48cb555

```suggestion
    LogInstance().m_file_path = AbsPathForConfigVal(args.GetPathArg("-debuglogfile", fs::path{DEFAULT_DEBUGLOGFILE}));
```